### PR TITLE
add missing fields for Card type

### PR DESCRIPTION
**Pull request by favadi**

### CODE REVIEW

File: v1/card/card.gunk [hunk 1]
```diff
@@ -68,26 +68,35 @@ type Card struct {
 	// Account is the ID of the account associated with the card.
 	Account string `pb:"2" json:"account"`
 
+	// CardNumber is the unique identify number of each card.
```
Comment (tsnewaz) [inline v1/card/card.gunk]: s/identify/identification

also, I assume it's okay to change the pb order below?

File: v1/types/types.gunk [hunk 2]
```diff
@@ -348,51 +348,54 @@ type Profile struct {
...
 
 	// Date of birth of dependents
-	DobOfDependents []time.Time `pb:"19" json:"dob_of_dependents"`
+	DobOfDependents []time.Time `pb:"20" json:"dob_of_dependents"`
 
 	// Credit rating
-	CreditRating CreditRating `pb:"20" json:"credit_rating"`
+	CreditRating CreditRating `pb:"21" json:"credit_rating"`
 
 	// Credit Limit
```
Comment (ghazninattarshah) [inline v1/types/types.gunk:375]: please avoid changing the order/number/pb field position. it won't be backward compatible.

**Review by ghazninattarshah — CHANGES_REQUESTED** [review]
Try to re-generate only the types proto bindings using gunk command.

File: v1/card/card.gunk [hunk 3]
```diff
@@ -68,26 +68,35 @@ type Card struct {
...
 	// Status is the status of the card.
-	Status types.CardStatus `pb:"8" json:"status"`
+	Status types.CardStatus `pb:"9" json:"status"`
 
 	// AccessStatus is the access status of the card.
-	AccessStatus types.CardAccessStatus `pb:"9" json:"access_status"`
+	AccessStatus types.CardAccessStatus `pb:"10" json:"access_status"`
+
+	// Specify when payments are due on money borrowed
+	AmountDue types.Amount `pb:"11" json:"amount_due"`
```
Comment (tsnewaz) [inline v1/card/card.gunk]: should the comment start with `AmountDue`?
e.g. // AmountDue specifies ...

also nit: end with punctuation to be consistent

File: v1/card/card.gunk [hunk 4]
```diff
@@ -137,17 +146,20 @@ type Result struct {
 
 // CreateCardRequest is the request data to create new card
 type CreateCardRequest struct {
+	// CardNumber is the unique identify number of each card.
```
Comment (tsnewaz) [inline v1/card/card.gunk]: s/identify/identification

**Review by tsnewaz — APPROVED** [review]
LGTM, just some nits

File: v1/card/card.gunk (hunk 3 above)
Comment (tsnewaz) [reply v1/card/card.gunk]: also the comment seems to imply a timestamp, probably worth a closer look

File: v1/card/card.gunk (hunk 1 above)
Comment (ghazninattarshah) [reply v1/card/card.gunk]: No, it won't be backward compatible if we change the order.

Comment (favadi) [reply v1/card/card.gunk]: > s/identify/identification

fixed.

File: v1/card/card.gunk (hunk 3 above)
Comment (favadi) [reply v1/card/card.gunk]: I just copied and paste the amount due in account :D,m will fix.

Comment (favadi) [reply v1/card/card.gunk]: fixed.

**Review by ghazninattarshah — APPROVED** [review]

File: v1/types/types.gunk (hunk 2 above)
Comment (favadi) [reply v1/types/types.gunk:375]: as discussed, we agree that the field position changing at this stage is acceptable.

**Comment by favadi** [comment]
> Try to re-generate only the types proto bindings using gunk command.

We still need to generate everything to have the new field in types/types.gunk added to docgen output of all consumers.

**Review by jwangsadinata — APPROVED** [review]
LGTM.

**Pull request merged**